packages/debug/src/node/vscode/vscode-debug-adapter-contribution.ts

```diff
@@ -14,6 +14,8 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
+/* eslint-disable @theia/localization-check */
```

Comment:
Perhaps add a second commit fixing these? it looks like it's mainly suggestions to use `byDefault` in place of the keyed version. For that violation, it might make sense to add a fixer to the plugin.

Reply:
Well, this file is a weird case anyway. This file uses it's own "implementation" (if one can say so), since it's in the `node` directory and has no actual localizations available:

https://github.com/eclipse-theia/theia/blob/89b8dd1ab7daa5dbead588ae30a02fc8f15ad595/packages/debug/src/node/vscode/vscode-debug-adapter-contribution.ts#L27-L31

This code has been there for quite a long time, or at least way before I started working on i18n in Theia. Actually, looking further into it, nothing that's exported in this file is referenced somewhere else. It seems to be dead code from before the vscode-extension time of Theia. I would remove this in a separate PR.